app/views/kaui/tag_definitions/_form.html.erb

```diff
@@ -19,3 +38,35 @@
       </div>
     </div>
 <% end %>
+
+<%= javascript_tag do %>
+
+    function new_object_type() {
+        var max_idx = $.map($('[id^=delete_object_type_]'), function(e, i) {
+            return e.id.split("delete_object_type_")[1] }
+        ).sort(function(a, b) {
+            return b - a
+        })[0];
+
+        var row = $('#object_types>div:last').clone(true);
```

Comment:
Could we update the options after cloning and remove the values already selected? In the end, the applicable object types list is really a set (unique values), so we shouldn't have any duplicate.